app.js

```diff
@@ -0,0 +1,60 @@
+var projectData = [
+  {
+    title: 'About Me',
+    preview: 'https://placekitten.com/300/300',
+    previewUrl: 'https://github.com/aer03280/about_me',
+    dateCreated: '2016-18-10',
+    description: '<p>This is a site I built with a get-to-know Ashley game.</p>',
+  },
+  {
+    title: 'Trip Planner',
+    preview: 'https://placekitten.com/300/300',
+    previewUrl: 'https://github.com/shortaj/Group_Project-Trip-Planner',
+    dateCreated: '2016-09-11',
+    description: '<p>This application was created as a group assignment. It helps you select a destination for your next vacation.</p>',
+
+  },
+  {
+    title: 'The Cookie Stand',
+    preview: 'https://placekitten.com/300/300',
+    previewUrl: 'https://github.com/aer03280/cookie-stand',
+    dateCreated: '2016-24-10',
+    description: '<p>I created this site as a class assignment for a fictitious cookie shop.</p>',
+  }
+
+];
+
+var arts = [];
+
+function Articles (projects) {
```

Comment:
Good convention for constructors is to have singular names, as they represent what an instance will become.